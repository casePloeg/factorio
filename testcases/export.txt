run_script "/Users/caseploeg/factorio/factorio-cli/scripts/startup.txt" > /dev/null
set debug true
alias create lprod prod "|" less
alias create prodl lprod
alias create raw prod "|" grep -E "(ore|plate)"
alias create init @ scripts/factory.txt
alias create export history -s ">" export.txt
mine iron-ore 20
mine stone 50
place stone-furnace iron-plate
place burner-mining-drill iron-ore
next 2
craft burner-mining-drill 3
craft stone-furnace 2
place burner-mining-drill iron-ore 3
place stone-furnace iron-plate 2
next 2
craft burner-mining-drill 2
place burner-mining-drill copper-ore
place burner-mining-drill stone
next 2
craft burner-mining-drill 2
place burner-mining-drill copper-ore 2
next 2
craft automation-science-pack 10
next 2
inventory
place stone-furnace iron-plate 2
craft stone-furnace 3
place stone-furnace iron-plate 2
place stone-furnace copper-plate
next 2
craft automation-science-pack 10
research automation
next 2
craft burner-mining-drill 2
craft assembling-machine-1 3
place assembling-machine-1 iron-gear-wheel
place assembling-machine-1 automation-science-pack
place assembling-machine-1 automation-science-pack
inventory
next 2
inventory
save ./expected/export.txt
