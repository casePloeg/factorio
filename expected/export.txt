{"assemblers": {"automation-science-pack:assembling-machine-1": 2, "iron-gear-wheel:assembling-machine-1": 1}, "current_items": {"assembling-machine-1": 0, "automation-science-pack": 24, "burner-mining-drill": 2, "copper-cable": 1, "copper-ore": 409, "copper-plate": 88, "iron-gear-wheel": 96, "iron-ore": 0, "iron-plate": 591, "stone": 182, "stone-furnace": 0, "water": 1000000000}, "current_recipes": ["assembling-machine-1", "automation-science-pack", "boiler", "burner-inserter", "burner-mining-drill", "copper-cable", "copper-plate", "electric-mining-drill", "electronic-circuit", "firearm-magazine", "inserter", "iron-chest", "iron-gear-wheel", "iron-plate", "iron-stick", "lab", "light-armor", "long-handed-inserter", "offshore-pump", "pipe", "pipe-to-ground", "pistol", "radar", "repair-pack", "small-electric-pole", "steam-engine", "stone-brick", "stone-furnace", "transport-belt", "wooden-chest"], "current_tech": ["automation"], "furnaces": {"copper-plate:stone-furnace": 1, "iron-plate:stone-furnace": 5}, "game_time": 1143.75, "limited_items": {}, "miners": {"copper-ore:burner-mining-drill": 3, "iron-ore:burner-mining-drill": 4, "stone:burner-mining-drill": 1}}